### DISCUSSION TITLE
How to make the bot send images?

### DISCUSSION THREAD

**Discussion by LinkTheCoder** [tjtanjin/react-chatbotify#61] [Q&A] [answered]
Hi, I'm new to react-chatbotify! 
I'm searching for a way to make the bot able to send an image as reply 😅

**Comment by tjtanjin** [comment, marked as answer]
> Hi, I'm new to react-chatbotify! 
> 
> I'm searching for a way to make the bot able to send an image as reply 😅

Hey @LinkTheCoder! For sending of images, you can consider using the `render` attribute as described here: https://react-chatbotify.tjtanjin.com/docs/api/attributes

An example of render component can also be found here (although the example shows a button instead): https://react-chatbotify.tjtanjin.com/docs/examples/render_component

Let me know if you need further clarifications! 😊
